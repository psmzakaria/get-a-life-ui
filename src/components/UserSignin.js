import React from "react";
import { Button, Form, Divider } from "semantic-ui-react";

const UserSignin = props => (
  <div className="ui inverted segment">
    <div className="ui inverted form">
      <h1>Sign In Page</h1>
      <Form>
        <Form.Field>
          <label>Email</label>
          <input placeholder="Email" />
        </Form.Field>
        <Form.Field>
          <label>Password</label>
          <input placeholder="Password" />
        </Form.Field>
        <Button fluid color='blue' type="submit">Sign In</Button>
      </Form>
      <Divider />
      <div>
        <p className="text-align-center">
          Don't have an account?
          <a className="cursor-pointer"
            onClick={() => props.loadComponent("Signup")}
          >
            {` `} Sign Up Here
          </a>
        </p>
      </div>
    </div>
  </div>
);

export default UserSignin;
