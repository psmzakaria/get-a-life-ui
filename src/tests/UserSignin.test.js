import React from "react";
import UserSignin from "./../components/UserSignin";

import ShallowRenderer from "react-test-renderer/shallow";

it("renders the sign in page", () => {
  const renderer = new ShallowRenderer();
  renderer.render(<UserSignin />);
  const output = renderer.getRenderOutput();
  expect(output).toMatchSnapshot();
});
