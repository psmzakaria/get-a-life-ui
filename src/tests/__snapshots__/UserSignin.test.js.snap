// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the sign in page 1`] = `
<div
  className="ui inverted segment"
>
  <div
    className="ui inverted form"
  >
    <h1>
      Sign In Page
    </h1>
    <Form
      as="form"
    >
      <FormField>
        <label>
          Email
        </label>
        <input
          placeholder="Email"
        />
      </FormField>
      <FormField>
        <label>
          Password
        </label>
        <input
          placeholder="Password"
        />
      </FormField>
      <Button
        as="button"
        color="blue"
        fluid={true}
        role="button"
        type="submit"
      >
        Sign In
      </Button>
    </Form>
    <Divider />
    <div>
      <p
        className="text-align-center"
      >
        Don't have an account?
        <a
          className="cursor-pointer"
          onClick={[Function]}
        >
           
           Sign Up Here
        </a>
      </p>
    </div>
  </div>
</div>
`;
